test/resources/DataFormats.py

```diff
@@ -68,6 +68,17 @@ def _merge(dest, addition):
     return dest
 
 
+def _replace(dest, replacement):
+    """Implements data structure field replacement
+
+    The dest value is modified in place by replacing all
+    the values found in replacement parameter.
+    """
+    for key, value in replacement.items():
+        dest[key] = value
```

Comment:
Can we use https://docs.python.org/3/library/stdtypes.html#dict.update or `d | other` or `d |= other` instead?

Reply:
Done